cypress/e2e/trigger_builder.cy.js

```diff
@@ -24,7 +24,10 @@ const triggerOperatorToLabel = {
 };
 
 const setupTriggerEditorFromSource = (trigger) => {
-  cy.get('#triggerSource').scrollIntoView().paste(JSON.stringify(trigger));
+  cy.window().then((win) => {
+    const editor = win.monaco.editor.getModels()[0];
+    editor.setValue(JSON.stringify(trigger, null, 2));
```

Comment:
those 2 functions (getModels() and setValue) are used quite often.
We could wrap them in a utility function inside cypress.
https://github.com/astarte-platform/astarte-dashboard/blob/master/cypress/support/e2e.js

```javascript
Cypress.Commands.add('pasteJsonIntoEditor', ({ json_object }) => {
  cy.window().then((win) => {
    const editor = win.monaco.editor.getModels()[0];
    editor.setValue(JSON.stringify(json_object, null, 2));
  });
});
```